### DISCUSSION TITLE
Welcome to LASER Discussions!

### DISCUSSION THREAD

**Discussion by pratyushasharma** [pratyushasharma/laser#10] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by bupt-yiwo** [comment]
In order to increase the speed of running the dataset, I have changed the single batch of the original code to 64 and 128 batches, and found that there is a significant drop in the effectiveness of the modified model, is there any solution for this?

**Comment by anuriita** [comment]
I installed the packages according to requirements.txt file but facing issue with the tokenizers and transformer versions. You have specified the transformer version but could you list the version of tokenizers that will work. I tried 0.10. 0.14 and 0.19 but it is giving error on each version.

Error - ImportError: tokenizers>=0.14,<0.15 is required for a normal functioning of this module, but found tokenizers==0.10.3
